service/src/main/java/io/camunda/service/search/query/DecisionRequirementsQuery.java

```diff
@@ -74,7 +95,8 @@ protected Builder self() {
     public DecisionRequirementsQuery build() {
       filter = Objects.requireNonNullElse(filter, EMPTY_FILTER);
       sort = Objects.requireNonNullElse(sort, EMPTY_SORT);
-      return new DecisionRequirementsQuery(filter, sort, page());
+      resultConfig = Objects.requireNonNullElse(resultConfig, EXCLUDE_XML_RESULT_CONFIG);
```

Comment:
❓ I might have missed it: Do we test this default behavior explicitly already, i.e. there is no XML returned although it exists in the entity if we don't explicitly include the `xml` attribute?

Reply:
zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/DecisionRequirementsQueryControllerTest.java asserts that we always set the exclude filter on `xml` attribute

Checking that the `xml` attribute is actually filtered out in the result, require an integration test with real ES instance, however we cannot do it with client integration tests as the field is not part of the API
I will check how we can do that